src/components/Comment/CommentStyles.js

```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled';
+
+export const CommentWrapper = styled.div`
+  background-color: #fcfcfc;
+  margin: 8px 0;
+  padding: 16px;
+  transition: box-shadow 0.1s ease-in;
+  border-radius: 4px;
+  color: #717171;
```

Comment:
Better define general color constants for whole project. And use them in styles.